compiler/lib/stdlib.ml

```diff
@@ -986,6 +1022,10 @@ module String = struct
         | _ -> false
     in
     loop (length b - 1) b 0
+
+  let fold_left ~f ~init s = Bytes.fold_left ~f ~init (Bytes.unsafe_of_string s)
```

Comment:
`Bytes.unsafe_of_string` is difficult to reason about. Even if it is correctly used here, I'd prefer to not have `Bytes.unsafe_of_string` in the codebase. You can just copy the implementation from Bytes.

Reply:
Done